migration/transitioning.rst

```diff
@@ -136,6 +136,42 @@ When joining tables make sure to filter by tenant id. For instance here is how t
      AND l.store_id='8c69aa0d-3f13-4440-86ca-443566c1fc75'
      AND p.store_id='8c69aa0d-3f13-4440-86ca-443566c1fc75'
 
+Validating Query Migration
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+With large, and complex application code-bases, certain queries generatedy by
+the application can often be overlooked, and thus won't have a tenant_id filter
+on them. Citus' parallel executor will still execute these queries successfully,
+and so, during testing, these queries remain hidden since the application still
+works fine. However, if a query doesn't contain the tenant_id filter, Citus's
```

Comment:
The docs have been using `Citus'` rather than `Citus's`. Just did some quick research about it and there is disagreement by various sources about which is the standard. Some people say to use `s'` for plural nouns ending in s and `'s` when the nouns are singular even if they end in s. Other places say always use `s'` for nouns ending in s (but not double-s).

I'd say stick with `Citus'` for consistency